design-system/packages/core/src/wrapHandlers.ts

```diff
@@ -0,0 +1,12 @@
+export const wrapHandlers = <EventType extends { defaultPrevented: boolean }>(
```

Comment:
Also don't think we're using this?

Reply:
yeah looks like residual from previous work, removing